### ISSUE TITLE
Update Schema - add updatedAt colums

### ISSUE THREAD

**Issue by d-username** [boolean-uk/team-dev-server#116]
Task description:

--> add updatedAt columns on schema where it's needed
--> when adding these columns we DON'T want to reset the existing table in the DB
--> we used "?" to make the columns optional ---> after Ed's advise we want to insert values to the new fields rather than using "?"

**Comment by dearshrewdwit** [comment]
From Heroku logs:
```
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "oywzjxsx", schema "prisma" at "manny.db.elephantsql.com:5432"
6 migrations found in prisma/migrations
Error: P3009
migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve
The `20220624093738_updating_models_created_at_updated_at` migration started at 2022-06-24 13:47:32.526846 UTC failed
```

**Comment by dearshrewdwit** [comment]
I've reset data in the tables, and then reran the deploy process to successfully run the new migration adding the columns to the tables.

tl;dr I've avoided the problem for you of updating data in the tables - all data will now have those fields.

**Comment by dearshrewdwit** [comment]
To do
- generate a migration removing the optional qualifier on the `updatedAt field`
